app/controllers/api/did_not_votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class API::DidNotVotesController < API::RestfulController
+
+  def create
+    raise NotImplementedError.new
+  end
+
+  def update
+    raise NotImplementedError.new
+  end
+
+  def destroy
+    raise NotImplementedError.new
+  end
+
+  private
+
+  def visible_records
+    load_and_authorize :motion
+    @motion.did_not_votes.order(:id)
```

Comment:
not a real good reason at this stage. was imagining that when we paginate it we need an order